_sections/30-bindings.md

```diff
@@ -101,7 +101,7 @@ A subset of [XPath 1.0 functions](https://www.w3.org/TR/1999/REC-xpath-19991116/
 
 | function                                  | description |
 |-------------------------------------------|------|
-<a id="fn:concat" href="#fn:concat">`concat(* arg*)`</a> | Deviates from [XPath 1.0](https://www.w3.org/TR/1999/REC-xpath-19991116/#function-concat) in that it may contain _1 argument_ and that all arguments can be _nodesets_ or strings. It concatenates all string values and _all node values_ inside the provided nodesets.
+<a id="fn:concat" href="#fn:concat">`concat(string arg*)`</a> | Deviates from [XPath 1.0](https://www.w3.org/TR/1999/REC-xpath-19991116/#function-concat) in that it may contain _1 argument_ and that all arguments can be _nodesets_ or strings. It concatenates all string values and _all node values_ inside the provided nodesets.
```

Comment:
Would this be clearer as `string|noseset arg*`?

Reply:
Yes, I think so! I can add that to the next xpath-functions PR.

Reply:
This isn’t merged here so you can just add it here and then merge!

Reply:
Though I guess this could also fall under the “anything goes” usage. Either way, you can make the final call and merge.

Reply:
Though I guess this could also fall under the “anything goes” usage. Either way, you can make the final call and merge.

Reply:
Ah thanks. I'll merge to save having to fix conflicts with the next one (coming very soon).